answer/56miyano/JavaScript/10_3sec.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <input type="button" value="押すな押すな" onclick="push()">
+    <script type="text/javascript">
+    function push() {
+            sanbyou = setTimeout ( timer , 3000);
```

Comment:
nits: 

```suggestion
            setTimeout(timer, 3000);
```

1. 使わないなら変数に代入しなくて良さそう
2. 関数呼び出しの丸括弧は関数名の直後に書きましょう(スペースはいらない)